packages/xstate-wallet/src/store/memory-channel-storage.ts

```diff
@@ -6,13 +6,18 @@ import {
   hashState,
   getStateSignerAddress
 } from '@statechannels/nitro-protocol';
-import {joinSignature, splitSignature} from 'ethers/utils';
-
-interface ChannelStorage {
-  myIndex: number;
-  channelConstants: ChannelConstants;
-  stateVariables: Record<string, StateVariables>;
-  signatures: Record<string, (string | undefined)[]>;
+import {splitSignature, joinSignature} from 'ethers/utils';
+
+export interface ChannelStorage {
```

Comment:
I would not reuse the name `ChannelStorage`. Tom is using it in his PR and it's also in the Solidity code.